packages/theming/apple-theme/src/appleTheme.ts

```diff
@@ -0,0 +1,30 @@
+import { Spacing, Theme } from '@fluentui-react-native/theme-types';
+import { getFluentUIApplePalette } from './fluentAppleColors';
+import { getAppleSemanticPalette } from './applePlatformColors';
+
+import { paletteFromAppleColors } from './appleTheme.colors';
+
+import { appleTypography } from './appleTheme.typography';
+
+export function appleSpacing(): Spacing {
+  return { s2: '4px', s1: '8px', m: '16px', l1: '20px', l2: '32px' };
```

Comment:
Where do these values come from/how are they used?

Reply:
These values are copied from the default theme. AFAIK, they aren't really used anywhere yet, though @falselobster can correct me if I'm wrong. 

We align to a a 4px grid anyway, so these felt fine...

Reply:
follow up with @rdeassis if spacing has already been tokenized in Apple. I think font and colors are but not sure about spacing...

Reply:
Looking [here](https://github.com/microsoft/fluentui-apple/blob/88073f40a14fb15af56688ab6b54dad91fb5d47d/tools/sgen/input/FluentUIStyle.yml#L147), spacing is defined as such:

```
Spacing:
  none: 0pt
  xxxSmall: 2pt
  xxSmall: 4pt
  xSmall: 8pt
  small: 12pt
  medium: 16pt
  large: 20pt
  xlarge: 24pt
  xxlarge: 36pt
  xxxlarge: 48pt
  xxxxlarge: 72pt
  ````
  
  It's not... exactly clear how these two line up but really the shape of the Theme interface should just evolve to match the design token pipeline that we have so I don't have to guess these equivalences in the future..

Reply:
Yea I think that's an accurate assessment.  This interface may be sort of a prototype/placeholder in absence of a fully spec'd out token pipeline.

Reply:
I changed this to match the type ramp from FluentUI Apple vNext (assuming s2 corresponds to xSmall, etc). This still isn't used anywhere, but at least it matches :D